src/Widgets/PreferencesDialog.vala

```diff
@@ -19,6 +19,45 @@
 * Authored by: Felipe Escoto <felescoto95@hotmail.com>
 */
 
+/**
+ *  The purpose of this class is to easily create a new Switchbox for the
+ *  PreferencesDialog class.
+ */
+private class ENotes.PreferencesDialogSwitchBox {
+
+    private Gtk.Box container;
+    private Gtk.Switch switchbox;
+    private Gtk.Label  label;
+
+    public Gtk.Box SwitchBox { get { return this.container; } }
+    public Gtk.Label SwitchBoxLabel { get { return this.label; } }
```

Comment:
variable_names should be cased_like_this 

https://elementary.io/docs/code/reference#variable-names-class-names-function-names